docs/components/store.md

```diff
@@ -23,7 +23,7 @@ config:
   bucket: example-bucket
 ```
 
-In general about 1MB of local disk space is required per TSDB block stored in the object storage bucket.
+In general, an average of 6 MB of local disk space is required per TSDB block stored in the object storage bucket. It is for the precomputed index, which includes symbols and postings offsets. The size of this precomputed index can go up to 30 MB per block.
```

Comment:
Thanks, but is it 6MB or 36MB of local disk in the end? (:

Reply:
What I understood from the slack conversation was that the space consumption is 6MB in average case but can go up to 30MB in extreme cases. Please do let me know if I got on the wrong track somewhere. :)

Reply:
Sorry for lag - yes it varies (: Let's mention this in docs then. What about:

```
In general, an average of 6 MB of local disk space is required per TSDB block stored in the object storage bucket, but high cardinality blocks with large label set it can go even to 30MB and more. It is for the pre-computed index, which includes symbols and postings offsets as well as metadata JSON.
```

Reply:
Thank you for the help! :)
I have made the recommended change.